samples/bluetooth/iso_time_sync/README.rst

```diff
@@ -154,7 +154,7 @@ The sample code is divided into multiple source files, which makes it easier to
     The very first SDU is provided without a timestamp, because the timestamp is not known at this point in time.
   * **LED1** is configured to toggle synchronously with the **LED1** on all the receivers.
     The toggle time is determined by the TX timestamp and defined relative to the synchronization reference on the receiver.
-  * If the :kconfig:option:`LED_TOGGLE_IMMEDIATELY_ON_SEND_OR_RECEIVE` Kconfig option is enabled, **LED2** is toggled right before sending the SDU.
+  * If the ``CONFIG_LED_TOGGLE_IMMEDIATELY_ON_SEND_OR_RECEIVE`` Kconfig option is enabled, **LED2** is toggled right before sending the SDU.
```

Comment:
I think for all such instances, we have to use the format like :ref:`CONFIG_LED_TOGGLE_IMMEDIATELY_ON_SEND_OR_RECEIVE <CONFIG_LED_TOGGLE_IMMEDIATELY_ON_SEND_OR_RECEIVE>` that in turn would require a section like https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/templates/sample_README.html#configuration-options where such options are defined.
```suggestion
  * If the ``CONFIG_LED_TOGGLE_IMMEDIATELY_ON_SEND_OR_RECEIVE`` Kconfig option is enabled, **LED2** is toggled right before sending the SDU.
```